mule-extensions-api-persistence/src/test/java/org/mule/runtime/extension/api/persistence/ExtensionModelPersistenceTestCase.java

```diff
@@ -168,15 +164,15 @@ public void validateCustomTypeAnnotations() throws IOException
     {
         MetadataType complexType = deserializedExtensionModel.getOperationModels().get(0).getParameterModels().get(1).getType();
         assertThat(complexType, instanceOf(ObjectType.class));
-        assertThat(getSingleAnnotation(complexType, TypeAliasAnnotation.class).isPresent(), is(true));
-        assertThat(getSingleAnnotation(complexType, TypeAliasAnnotation.class).get().getValue(), is(ComplexFieldsType.ALIAS));
+        assertThat(complexType.getAnnotation(TypeAliasAnnotation.class).isPresent(), is(true));
```

Comment:
That's correct. Only changed the `getAnnotation` interface and the DescriptionAnnotation (which it is indeed serialized differently but is not used in this test).